mix.exs

```diff
@@ -58,6 +58,7 @@ defmodule Liveman.MixProject do
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:phoenix_live_dashboard, "~> 0.4"},
       {:plug_cowboy, "~> 2.0"},
+      {:scrivener_list, "~> 2.0"},
```

Comment:
Why picking this library over https://github.com/elixirdrops/kerosene as it was suggested by Micky in this [Slack thread](https://nimble-co.slack.com/archives/C0112R61J3W/p1623640061002400?thread_ts=1623639276.000700&cid=C0112R61J3W)

Reply:
The `kerosene` library is for `Ecto`. But here we are not storing any data into the database(we are just fetching surveys data from the file, assign them to the `embedded_schema` and expose the list with `jsonapi`  just like how our rails survey app does). 

We get here only survey `list` and the `scrivener_list` library is just for `list` pagination. like bellow

```elixir
[ "Elixir", "Ruby", "Go", "JavaScript", "PHP",  "Perl", "Python"] |> Scrivener.paginate(%{})
```

Whst do you think?

Reply:
That looks fine 😁  I was just wondering about it 👍